genienlp/tasks/almond/__init__.py

```diff
@@ -233,7 +234,10 @@ class AlmondDialogueNLU(BaseAlmondTask):
     state of the conversation)
     """
     def _is_program_field(self, field_name):
-        return field_name in ('answer', 'context')
+        if self.append_question_to_context_too:
+            return field_name == 'answer'
+        else:
+            return field_name in ('answer', 'context')
```

Comment:
You don't need this, cause you'll get "question" as the field name for the concatenated field?

Reply:
Yes, it is a remnant of another approach. I'll remove it.